src/main/java/uk/gov/ons/ctp/response/casesvc/representation/CaseEventCreationRequestDTO.java

```diff
@@ -34,4 +35,6 @@
   private String createdBy;
 
   private String subCategory;
+
+  private HashMap<String, String> metadata;
```

Comment:
You should use the interface `Map` instead of the concrete implementation `HashMap`.